moulinette/utils/filesystem.py

```diff
@@ -77,6 +80,28 @@ def read_yaml(file_path):
     return loaded_yaml
 
 
+def read_toml(file_path):
+    """
+    Safely read a toml file
+
+    Keyword argument:
+        file_path -- Path to the toml file
+    """
+
+    # Read file
+    file_content = read_file(file_path)
+
+    # Try to load toml to check if it's syntaxically correct
```

Comment:
```suggestion
    # Try to load toml to check if it's syntactically correct
```

Unsure if "syntaxically" is a new cool way to say this though :)

Reply:
In french we have "syntaxique" for "syntactic", like "connexion" for "connection" and a few others like that, it's very confusing --''.